packages/calypso-products/src/plans-list.tsx

```diff
@@ -1796,6 +1862,14 @@ export const PLANS_LIST: Record< string, Plan | JetpackPlan | WPComPlan > = {
 		getStoreSlug: () => PLAN_JETPACK_SECURITY_T2_YEARLY,
 		getPathSlug: () => 'security-1tb-yearly',
 		getProductId: () => 2019,
+		getWhatIsIncluded: () => [
+			translate( 'Real-time backups as you edit' ),
+			translate( '1TB (1,000GB) of cloud storage' ),
+			translate( '1-year activity log archive' ),
+			translate( 'Unlimited one-click restores from the last 1 year' ),
```

Comment:
In these "What's included" items for SECURITY_T2 and also BACKUP_T2 products, there are some parts of it that should be **bold** type.  For example:

- **1TB (1,000GB)** of cloud storage
- **1-year** activity log archive
- Unlimited one-click restores from the last **1 year**

As shown on the Figma design here: m0y1jHTuDpvMPkFsOJBoNP-fi-3557%3A123824